hlxsites/merative2
# fix(article-detail): add new design changes for publication placement

## Issue

Fixes https://github.com/hlxsites/merative2/issues/236

## Description

**Changed**

- This PR implements the new design treatment for the publication info within Blog detail pages.
- Keep the author's position below the image, but move "Published on" and "Read Time" to be above the image and right below the tags.

## Design Specs

- Figma Link - https://www.figma.com/file/vw24IPXXt4vCh9BNumka9A?node-id=5988:24513&mode=design#481441510

## Test URLs

![Screen Shot 2023-06-29 at 10 22 32 AM](https://github.com/hlxsites/merative2/assets/1815714/e9dc0389-6885-4aac-ae2b-af97f77578b1)

- Before (Changes from `main`): https://main--merative2--hlxsites.hlx.page/drafts/putra/test-blog-detail-page
- After (Changes from this PR): https://fix-760-published-date--merative2--proeung.hlx.page/drafts/putra/test-blog-detail-page
  
## Testing Instruction

- Visit this example page - https://fix-760-published-date--merative2--proeung.hlx.page/drafts/putra/test-blog-detail-page
- Ensure that the changes align with the design specs for all of the support breakpoints
